From dca415d6d834825fb59fd04338aac21429d4f0e5 Mon Sep 17 00:00:00 2001
From: Matthias Runge <mrunge@redhat.com>
Date: Wed, 14 Oct 2015 11:40:03 +0200
Subject: [PATCH] respect webroot setting in containers table

Change-Id: I847b8e4589a7b3122bb23431f1d88038dc10ae6c
Closes-Bug: #1505976
(cherry picked from commit 1296707d37abce5fcc7f1030f852875fbae9aff4)
---
 openstack_dashboard/dashboards/project/containers/tables.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/openstack_dashboard/dashboards/project/containers/tables.py b/openstack_dashboard/dashboards/project/containers/tables.py
index f58da71..637ebf3 100644
--- a/openstack_dashboard/dashboards/project/containers/tables.py
+++ b/openstack_dashboard/dashboards/project/containers/tables.py
@@ -13,6 +13,7 @@
 #    under the License.
 import logging
 
+from django.conf import settings
 from django import shortcuts
 from django import template
 from django.template import defaultfilters as filters
@@ -32,7 +33,8 @@ from openstack_dashboard.dashboards.project.containers import utils
 
 
 LOG = logging.getLogger(__name__)
-LOADING_IMAGE = '<img src="/static/dashboard/img/loading.gif" />'
+static_url = getattr(settings, 'STATIC_URL', '/static/')
+LOADING_IMAGE = '<img src="%s/dashboard/img/loading.gif" />' % static_url
 
 
 class ViewContainer(tables.LinkAction):
-- 
1.9.1

